README.md

```diff
@@ -19,3 +19,20 @@ Simlar to the [Ethereum Improvement Proposals (EIPs) repository](https://github.
 
 This index is manually maintained for now. \
 Please update the table whenever you add a new RPIP or update the status of an RPIP.
+
```

Comment:
Shall we get rid of the Index table entirely? Is there a point to a manual one given the jekyll site?

Reply:
Maybe instead of the table we'll have:

```markdown
For easy viewing of existing RPIPs, please see the [RPIP dashboard](https://rpips.netlify.app/).
Note that, for now, the dashboard must be manually rebuilt and published when the repo changes.
```